src/assets/HTMLAsset.js

```diff
@@ -92,6 +93,24 @@ class HTMLAsset extends Asset {
     return newSources.join(',');
   }
 
+  collectInlineStyleDependencies(inlineStyle) {
```

Comment:
This code is very hard to read. Could you try to add comments to make it easier to understand?

Reply:
just add it in the newest commit.